test/extended/operators/operators.go

```diff
@@ -167,6 +169,37 @@ var _ = g.Describe("[Feature:Platform][Smoke] Managed cluster should", func() {
 	})
 })
 
+var _ = g.Describe("[Feature:Platform] Managed cluster should", func() {
+	defer g.GinkgoRecover()
+
+	g.It("have operators on the cluster version", func() {
```

Comment:
`s/version/operator`

Reply:
?  cluster version is the version we want to have operator report

Reply:
the test reads `[Feature:Platform] Managed cluster should have operators on the cluster version` which threw me off a little... this is fine as-is